code/modules/unit_tests/closets.dm

```diff
@@ -12,7 +12,8 @@
 
 		// Copy is necessary otherwise closet.contents - immediate_contents returns an empty list
 		var/list/immediate_contents = closet.contents.Copy()
-
+		if (QDELETED(closet))
```

Comment:
Put this after the allocate, then write a comment specifying its for the emcloset

Reply:
Ok